tests/docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+services:
+  notifications:
+    build: "../."
+    expose:
+      - 443
+    environment:
+      DD_CONNECTOR_URL: "https://localhost:5000"
```

Comment:
Do we still need `DD_CONNECTOR_URL`?

Reply:
This is still in the notifications service, even though we probably don't need it any more.

Reply:
I've been able to completely strip-out the DD stuff without any negative effects.